dyld[18723]: Library not loaded: @executable_path/../Python3
  Referenced from: /Users/oskaripeltonen/Library/Application Support/pypoetry/venv/bin/python3
  Reason: tried: '/Users/oskaripeltonen/Library/Application Support/pypoetry/venv/bin/../Python3' (no such file), '/usr/local/lib/Python3' (no such file), '/usr/lib/Python3' (no such file)

Traceback:

  File "<stdin>", line 872, in main
  File "<stdin>", line 505, in run
